kitsune/kpi/static/kpi/js/components/Chart.es6.js

```diff
@@ -0,0 +1,182 @@
+/* globals d3:false, $:false, _:false */
+/* jshint esnext: true */
+
+export default class Chart {
+  constructor($container, options) {
+    let defaults = {
+      chartColors: ['#EE2820', '#F25743', '#F58667', '#F9B58B', '#FDE4AF', '#E3F1B6', '#AADA9F', '#72C489', '#39AD72', '#00975C'],
+      axes: {
+        xAxis: {
+          labels: [],
+          labelOffsets: { x: 15, y: 23 }
+        },
+        yAxis: {
+          labels: [],
+          labelOffsets: { x: 5, y: 23 }
+        },
+        getPosition: (position, axis, index) => {
```

Comment:
This style for functions works fine. You can also use another new syntax for functions in objects which feels more natural to me, like this:

```javascript
let adder = {
  value: 5,
  add(x) {
    return x + this.value;
  }
}
```

Use whichever you like better.